build/vs/BrainaliveCommands.h

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
```

Comment:
Please put everything Brainalive related in its own directory

Reply:
@dioptre please checkout to the latest commit I have updated my code with the latest version.